archunit/src/main/java/com/tngtech/archunit/core/importer/ImportOption.java

```diff
@@ -137,4 +149,16 @@ public boolean includes(Location location) {
             return !location.isArchive();
         }
     }
+
+    /**
+     * The {@code maven-compiler-plugin 3.10.0} fixed a bug that made {@code package-info.class}
+     * files hidden from {@code ArchUnit}.
+     * This is a {@link ImportOption} implementation for excluding {@code package-info.class} files.
+     */
+    final class DoNotIncludePackageInfoClasses implements ImportOption {
```

Comment:
Do you think we really need the suffix `Classes`? :thinking: Or would `DoNotIncludePackageInfos` be good enough? Just wondering what is consistent, because it's also called `DoNotIncludeTests` and not `DoNotIncludeTestClasses` :thinking:

Reply:
good point.
fixed in https://github.com/TNG/ArchUnit/pull/794/commits/90c126f6acba68e0602e316c5d576001ee5f20ce